README.md

```diff
@@ -33,13 +33,23 @@ import { DiscussionEmbed } from 'disqus-react';
             url: this.props.article.url,
             identifier: this.props.article.id,
             title: this.props.article.title,
+            language: 'zh_TW' //e.g. for Traditional Chinese (Taiwan)	
         }
     }
 />
 ```
 
 This component is limited to one instance in the DOM at a time and will handle updates to both the `config` and `shortname` props and reload appropriately with the new discussion thread.  
 
+#### `language` configuration
+
+This configuration overrides will load the Disqus embed in different languages on a per-page basis.
+
+- Language code can be found on: https://www.transifex.com/explore/languages/ 
+- The supported languages by disqus are available on https://www.transifex.com/disqus/disqus/
+
+> Note: Depending on the completeness of the translation for a given language, and the release of language support from disqus, the strings displayed on discussion thread UI may not be completely translated.
+
```

Comment:
This information is very helpful, but in an effort to keep or `README.md` as streamlined as possible I think it might be more appropriate to put this information in the [FAQ wiki page](https://github.com/disqus/disqus-react/wiki/Frequently-Asked-Questions).
I don't know if contributors are able to edit the wiki pages, but if not, I can transfer this information over after the PR is merged.

Reply:
Thanks for the feedback @tterb. I have checked the wiki of this repo and I don't think I have the write access to it. Feel free to remove the sections you think should be part of the wiki for better organization & clarity.

Reply:
I added an entry to the [FAQ wiki page](https://github.com/disqus/disqus-react/wiki/Frequently-Asked-Questions) that includes the provided usage information and Transifex links 👍

I also created a [Parameters wiki page](https://github.com/disqus/disqus-react/wiki/Parameters) page, where I hope to add additional usage information for the increasing number of props that are supported for each component.